RasterAU_AGD66_84_GDA94_2020DirInv.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+"""
+***************************************************************************
+    RasterPT_ETR89PTTM06DirInv.py
```

Comment:
@alexgleith 

you may want to change this to "RasterAU_AGD66_84_GDA94_2020DirInv.py"